src/worker.js

```diff
@@ -349,8 +422,8 @@
 
    ClientEngine.prototype.handleAPIRequest = function(portConduit, request) {
       var self = this;
-      var client = connect.core.getClient();
```

Comment:
Any reason this was changed from a clojure to a private variable?  connect.core.getClient() returns a singleton from core, so there is no overhead to doing this.